sass/themes/donate/donate-cr/variables/_colours.scss

```diff
@@ -1,3 +1,4 @@
 $colour-smoke-grey: #f0f0f0;
 $colour-mercury-grey: #e8e8e8;
 $colour-dark-purple: #29022c;
+$colour-blue: #4031ff;
```

Comment:
I think this is supposed to be the "royal blue" from the CR theme. We should be able to use those colours directly. The Royal blue is almost the same: $colour-royal-blue: #4032fe